kanidmd_web_ui/pkg/LICENSE.md

```diff
@@ -0,0 +1,374 @@
+Mozilla Public License Version 2.0
```

Comment:
We have the license at the project root? Is there a reason to duplicate it here?

Reply:
It makes wasm-bindgen happier

Reply:
found a license-file setting, which I put into 7b14dbe2 ... which just copies the file locally instead of referring to it 🙄